webapps/frontend/app/viewmodels/containers/albaosd.js

```diff
@@ -90,10 +90,10 @@ define([
                 generic.trySet(self.port, data, 'port');
                 generic.trySet(self.ips, data, 'ips');
                 generic.trySet(self.type, data, 'type');
-                if (data.hasOwnProperty('alba_backend_guid') && data.alba_backend_guid !== null) {
-                    self.albaBackendGuid(data.alba_backend_guid);
+                if (data.hasOwnProperty('claimed_by') && data.claimed_by !== null) {
```

Comment:
Isn't `claimed_by` an empty string when not linked to any Backend yet?

Reply:
It's null actually.
![image](https://user-images.githubusercontent.com/17570109/29812633-88a34ae4-8ca7-11e7-993e-77f7d374f007.png)